components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -1327,9 +1327,10 @@ func startContentInit(ctx context.Context, cfg *Config, wg *sync.WaitGroup, cst
 
 	fn := "/workspace/.gitpod/content.json"
 	fnReady := "/workspace/.gitpod/ready"
-	if _, err := os.Stat("/.workspace/.gitpod/content.json"); !os.IsNotExist(err) {
+	if _, err := os.Stat("/.workspace/.gitpod/pvc"); !os.IsNotExist(err) {
 		fn = "/.workspace/.gitpod/content.json"
-		log.Info("Detected content.json in /.workspace folder, assuming PVC feature enabled")
+		fnReady = "/.workspace/.gitpod/ready"
```

Comment:
we need to wait for ready file that comes from PVC layer, we should not use ready file that comes from `/workspace` since that file will be backed up into volume snapshot when it is created from pvc.
so this makes a clear separation between pvc and non-pvc code path.